# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit-test/tests/debug/Frame-onStack-07.js
# Commit: fa46d8caefde
# Full Hash: fa46d8caefdeea83b7a41ba3259114c13dd3b145
# Author: Logan Smyth <loganfsmyth@gmail.com>
# Date: 2019-12-09 09:50:39
# Regressor Bug: 1600204
# File Overlap Count: 1
# Description:
#   Bug 1600204 - Part 6: Support .older on suspended generator frames. r=jimb
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D55236
# ==============================================================================

diff -r ea1bb2f0fd39 -r fa46d8caefde js/src/jit-test/tests/debug/Frame-onStack-07.js
--- a/js/src/jit-test/tests/debug/Frame-onStack-07.js	Sun Dec 08 23:53:27 2019 +0000
+++ b/js/src/jit-test/tests/debug/Frame-onStack-07.js	Sun Dec 08 23:54:15 2019 +0000
@@ -45,7 +45,7 @@
     assertEq(poppedFrame.onStack, false);
     assertErrorMessage(() => poppedFrame.older,
                        Error,
-                       "Debugger.Frame is not on stack");
+                       "Debugger.Frame is not on stack or suspended");
 }
 
 for (let when = 0; when < 6; when++) {
